dask/array/core.py

```diff
@@ -2090,11 +2095,14 @@ def normalize_chunks(chunks, shape=None, limit=None, dtype=None,
     if not chunks and shape and all(s == 0 for s in shape):
         chunks = ((0,),) * len(shape)
 
+    if (shape and len(shape) == 1 and len(chunks) > 1 and
+            all(type(c) is not tuple for c in chunks)):
```

Comment:
Would it be ok to replace `type(c) is not tuple` with `isinstance(c, numbers.Number)`?  That way we'll probably continue doing something sensible if they pass a list like `chunks=[[1], [2], [3], [4]]`.  Obviously this isn't recommended, but new users can do all sorts of things :)

Reply:
``[[1], [2], [3], [4]]`` is  already covered by the unit tests. Anyway I changed it.